src/main/java/lotto/domain/Rank.java

```diff
@@ -0,0 +1,55 @@
+package lotto.domain;
+
+import java.lang.reflect.Array;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public enum Rank {
+    FIRST(2000000000, 6, false),
+    SECOND(30000000, 5, true),
+    THIRD(1500000, 5, false),
+    FOURTH(50000, 4, false),
+    FIFTH(5000, 3, false),
```

Comment:
```suggestion
    FIFTH(5_000, 3, false),
```

언더바를 이용하여 긴 숫자는 가독성을 높힐 수 있답니다 !